lib/features/pokemon_details/widgets/pokemon_details_overview.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:pokedex_async_redux/features/pokemon_details/widgets/widgets.dart';
+import 'package:pokedex_async_redux/api/model/model.dart';
+import 'package:pokedex_async_redux/widgets/widgets.dart';
+import 'package:flutter/material.dart';
+
+class PokemonDetailsOverview extends StatelessWidget {
+  const PokemonDetailsOverview({
```

Comment:
seems no need for this class anymore, just directly place these components in your `pokemon_details_page` 🙂